### PR TITLE
Contractor CSI Fix

### PR DESCRIPTION
**Pull request by NotSchwann**
AKA: Makes forensics singular.

- Fixes a minor typo in the Idris jobs datum, allowing Idris forensic technicians to spawn with proper equipment.

### CODE REVIEW

File: html/changelogs/neworiginalschwann - spellcheck.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Fixed a minor typo in the Idris CSI loadout datum. They should now spawn in with the proper ID and uniform."
+  - rscdel: "Changed 'forensics' to 'forensic' in datum."
```
Comment (NonQueueingMatt) [inline html/changelogs/neworiginalschwann - spellcheck.yml]: This isn't really needed as it's the same as the above. Just remove the second line.

File: code/game/jobs/faction/idris.dm [hunk 2]
```diff
@@ -53,7 +53,7 @@
 
 	titles_to_loadout = list(
 		"Security Officer" = /datum/outfit/job/officer/idris,
-		"Forensics Technician" = /datum/outfit/job/forensics/idris,
+		"Forensic Technician" = /datum/outfit/job/forensics/idris,
```
Comment (NonQueueingMatt) [inline code/game/jobs/faction/idris.dm]: Necropolis has this problem too - maybe fix it there as well?

File: html/changelogs/neworiginalschwann - spellcheck.yml (hunk 1 above)
Comment (NotSchwann) [reply html/changelogs/neworiginalschwann - spellcheck.yml]: Done.

File: code/game/jobs/faction/idris.dm (hunk 2 above)
Comment (NotSchwann) [reply code/game/jobs/faction/idris.dm]: Done but they still look like stereotypical bad guys, which I can't exactly change so it's whatever.

**Comment by NotSchwann** [comment]
!review !bugfix

**Comment by NotSchwann** [comment]
!bugfix

**Pull request merged**